src/main/java/com/extremelyd1/game/chat/ChatChannelController.java

```diff
@@ -0,0 +1,30 @@
+package com.extremelyd1.game.chat;
+
+import org.bukkit.entity.Player;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ChatChannelController {
+
+    public enum ChatChannel {
+        TEAM(), GLOBAL()
+    }
+
+    private static final ChatChannel DEFAULT_CHANNEL = ChatChannel.TEAM;
+
+    public Map<String, ChatChannel> playerChatChannels = new HashMap<>();
+
+    public void setPlayerChatChannel(Player player, ChatChannel channel) {
+        playerChatChannels.put(player.getUniqueId().toString(), channel);

```

Comment:
Is using the `String` representation of the UUID better than using the UUID object itself as a key in a HashMap? I believe UUID overrides the `hashCode` and `equals` methods, so it would be perfectly fine to use as a key.

Reply:
ah yeah it does override those methods, I wasn't sure if it was a safe key. Changed the key to the UUID.